src/mixins/filing-template-mixin.ts

```diff
@@ -100,7 +101,18 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
           rulesFileKey: this.getCreateRulesStep.docKey,
           rulesFileName: this.getCreateRulesStep.rulesDoc ? this.getCreateRulesStep.rulesDoc.name : null,
           rulesFileSize: this.getCreateRulesStep.rulesDoc ? this.getCreateRulesStep.rulesDoc.size : null,
-          rulesFileLastModified: this.getCreateRulesStep.rulesDoc ? this.getCreateRulesStep.rulesDoc.lastModified : null
+          rulesFileLastModified: this.getCreateRulesStep.rulesDoc
+            ? this.getCreateRulesStep.rulesDoc.lastModified : null,
+          rulesConfirmed: this.getCreateRulesStep.rulesConfirmed ? this.getCreateRulesStep.rulesConfirmed : false,
+          memorandumFileKey: this.getCreateMemorandumStep.docKey,
+          memorandumFileName: this.getCreateMemorandumStep.memorandumDoc
+            ? this.getCreateMemorandumStep.memorandumDoc.name : null,
+          memorandumFileSize: this.getCreateMemorandumStep.memorandumDoc
+            ? this.getCreateMemorandumStep.memorandumDoc.size : null,
+          memorandumFileLastModified: this.getCreateMemorandumStep.memorandumDoc
+            ? this.getCreateMemorandumStep.memorandumDoc.lastModified : null,
+          memorandumConfirmed: this.getCreateMemorandumStep.memorandumConfirmed
+            ? this.getCreateMemorandumStep.memorandumConfirmed : false
```

Comment:
You may be able to simplify here as this is a little tricky to process at first glance. 
Since most of these ternary are simple a means of short circuiting a particular value, you may get away with just using fallback values, which could lessen the code here. 

For example, `rulesFileLastModified: this.getCreateRulesStep.rulesDoc?.lastModified || null`
If you worried about `rulesDoc` not being present you can optional chain it, so in the above case, if rules doc exists it will apply `lastModified`, if it doesn't, it will fall back to null and prevent a typeError.

I imagine this approach could be use used for most scenarios above, give it a test though! :)

Reply:
good call...will give it a try

Reply:
cleaned things up.  looks a lot nicer now 😋

Reply:
nice!! Gj